io_scene_nif/nif_import.py

```diff
@@ -668,7 +668,6 @@ def import_mesh(self, niBlock,
             b_mesh = bpy.data.meshes.new(ni_name)
             # create mesh object and link to data
             b_obj = self.objecthelper.create_b_obj(niBlock, b_mesh)
-            # b_name = self.import_name(niBlock)
```

Comment:
Not sure if this was a fix me later situation.

Reply:
I'm still working out whether import_name() is needed at all. At least in its old form it no longer is. create_b_obj() takes care of all that is needed for object names.